habitat-lab/habitat/tasks/rearrange/actions/pddl_actions.py

```diff
@@ -38,7 +37,10 @@ def action_space(self):
             {
                 self._action_arg_prefix
                 + "pddl_action": spaces.Box(
-                    shape=(action_n_args,), low=-1, high=1, dtype=np.float32
+                    shape=(action_n_args,),
```

Comment:
What is the meaning of the action space in pddl? What does 0, np.finfo(np.float32).min, np.finfo(np.float32).max, represent?

Reply:
I worry about having the min and max of this action space be infinity.